README.md

```diff
@@ -8,114 +8,187 @@
     .SetExporter(dtExporter)
-    .SetPushInterval(TimeSpan.FromSeconds(pushIntervalInSecs))
+    .SetPushInterval(TimeSpan.FromSeconds(30))
     .Build());
 
-// get Meter, create a counter instrument and provide first data point
-var meter = MeterProvider.Default.GetMeter("MyMeter");
-var testCounter = meter.CreateInt64Counter("MyCounter");
+var meter = MeterProvider.Default.GetMeter("my_meter");
+var myCounter = meter.CreateInt64Counter("my_counter");
 var labels = new List<KeyValuePair<string, string>>()
 {
-    new KeyValuePair<string, string>("dimension-1", "value-1")
+    new KeyValuePair<string, string>("my_label", "value1")
 };
-testCounter.Add(defaultContext, 100, meter.GetLabelSet(labels));
+var defaultContext = default(SpanContext);
+
+myCounter.Add(defaultContext, 100, meter.GetLabelSet(labels));
 ```
 
-Optionally, it is possible to set additional properties on the `DynatraceExporterOptions` object:
+In addition to the `Url` and `ApiToken`, optional properties can be set on the `DynatraceExporterOptions` object, which are described in the [Configuration section](#configuration):
+
+- A `Prefix`, that is prepended to every metric key.
+- `DefaultDimensions`, which are added as dimensions to every exported metric
+- A toggle, `EnrichWithOneAgentMetadata`, which allows turning off the enrichment of metrics with host-specific information.
+    See [below](#dynatrace-api-endpoint) for more information.
 
 ```csharp
-var options = new DynatraceExporterOptions
+var exporterOptions = new DynatraceExporterOptions()
 {
-    // If URL and API token are not provided, the exporter will try to send metrics to the default OneAgent endpoint.
-    // However, it is also possible to specify Url and ApiToken here.
+    Url = "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest",
+    ApiToken = "YOUR_API_TOKEN",
+    Prefix = "metric.key.prefix",
+    DefaultDimensions = new List<KeyValuePair<string, string>>()
+    {
+        new KeyValuePair<string, string>("defaultDim1", "value1")
+        new KeyValuePair<string, string>("defaultDim2", "value2")
+    },
+    EnrichWithOneAgentMetadata = false,
+};
+```
 
-    // *** Additional (optional) parameters: ***
-    // A prefix that will be prepended to each exported metric name.
-    Prefix = "dynatrace.opentelemetry",
+## Example application
 
-    // Default dimensions are added to each exported metric as "key=value" pairs.
-    DefaultDimensions = new List<KeyValuePair<string, string>>{
-        new KeyValuePair<string, string>("key1", "value1"),
-        new KeyValuePair<string, string>("key2", "value2"),
-    },
+We provide an example command line application which exports metrics to Dynatrace.
+To run it use
 
-    // If a OneAgent is running on the same host as the application, it is possible to add host- and process-specific data as dimensions automatically.
-    OneAgentMetadataEnrichment = true,
-};
+```sh
+dotnet run --project src/Examples.Console/Examples.Console.csproj
```

Comment:
Ah I see that you list both, I think one option should do. The one from inside is probably better since it makes the commands you list further below more concise, currently they don't fit on the page.

Reply:
I just left the one with cd in.